squad-server/plugins/db-log.js

```diff
@@ -577,4 +626,44 @@ export default class DBLog extends BasePlugin {
       reviverSquadID: info.reviver ? info.reviver.squadID : null
     });
   }
+
+  async onPlayerConnected(info) {
+    if (!info || !info.player) return;
+
+    await this.startSession(info.player.steamID, info.player.name, info.time);
```

Comment:
This method is only called once so is there much point in having it? Just move the logic from the method here I think.